libwnbd/libwnbd.cpp

```diff
@@ -235,6 +235,80 @@ DWORD WnbdRemoveEx(
     return Status;
 }
 
+DWORD WnbdSetDiskSize(PWNBD_DISK Disk, UINT64 BlockCount) {
+    DWORD ErrorCode = ERROR_SUCCESS;
+    DWORD BytesReturned = 0;
+
+    LogDebug("Resizing device %s. New block count: %d",
```

Comment:
let's use an info message instead. also, we should also log the previous block count.